src/pages/robot/tsx/robot.tsx

```diff
@@ -731,12 +731,14 @@ export class Robot extends React.Component {
 
   executePoseGoal(pose: RobotPose) {
     this.switchToPositionMode();
+    this.stopExecution();
```

Comment:
I tried "Look to..." several times in a row and did not cause the wrist pitch to drop. I also tried manually moving the camera and it does not make the wrist pitch drop. Currently, the wrist pitch only drops on `2051` when I move the gripper, wrist, and arm. @hello-amal can you confirm if you see the same behavior on Stretch 3?

Reply:
That is expected behavior -- we know wrist pitch drops upon gripper, wrist, and arm movements ([`stretch_ros2`#140](https://github.com/hello-robot/stretch_ros2/issues/140)). My main concern was whether this PR adds more scenarios where the wrist pitch drops, because it added a "stop execution" call before movement recorder playback and "look at...". However, both you and I have verified that that is not the case.

I tried running click-to-pregrasp on `3030` and terminating it by commanding a single arm, wrist, or gripper move. The wrist pitch didn't fall. Which makes me think the falling issue has to do with repeated cancellation commands, not just one.